docs/upgrade/3.5.rst

```diff
@@ -113,6 +111,38 @@ Run::
 to apply the new migrations.
 
 
+***********************************
+Create a new django CMS 3.5 project
+***********************************
+
+On the Divio Cloud
+==================
+
+The Divio Cloud offers an easy way to set up django CMS projects. In the `Divio Cloud Control Panel
+<https://control.divio.com>`_, create a new django CMS project.
+
+Once it has been created, configure its *Addons*. You will need to specify the versions of two
+addons:
+
+* django CMS: select version ``3.5.0.1`` or newer from the *Beta* channel of Aldryn django CMS
+* django CMS Snippet: select ``2.0.0rc1`` or newer from the *Alpha* channel of django CMS Snippet
```

Comment:
this is no longer needed as snippet has been released and so will get installed with latest version